osu.Game/Overlays/Music/PlaylistList.cs

```diff
@@ -127,21 +125,27 @@ public BeatmapSetInfo SelectedSet
             public BeatmapSetInfo NextSet => (items.SkipWhile(i => !i.Selected).Skip(1).FirstOrDefault() ?? items.FirstOrDefault())?.BeatmapSetInfo;
             public BeatmapSetInfo PreviousSet => (items.TakeWhile(i => !i.Selected).LastOrDefault() ?? items.LastOrDefault())?.BeatmapSetInfo;
 
+            private InputState dragInputState;
+            private PlaylistItem draggedItem;
+
             protected override bool OnDragStart(InputState state)
             {
+                dragInputState = state;
                 draggedItem = items.FirstOrDefault(d => d.IsDraggable);
                 return draggedItem != null || base.OnDragStart(state);
             }
 
             protected override bool OnDrag(InputState state)
             {
+                dragInputState = state;
                 if (draggedItem == null)

```

Comment:
pretty sure this check is not required (`OnDrag` is only sent if returning true to `OnDragStart`)

Reply:
and returning true in `OnDragStart` does not necessarily imply that the little handle to the left of the items was dragged.

Reply:
why not? i could understand if we were a `ScrollContainer`, but how does this work?

Reply:
we are a `ScrollContainer`

Reply:
nested classes sure don't diff well